### DISCUSSION TITLE
2.0.0

### DISCUSSION THREAD

**Discussion by imrajdas** [litmuschaos/litmus#3117] [Releases]
<hr /><em>This discussion was created from the release <a href='https://github.com/litmuschaos/litmus/releases/tag/2.0.0'>2.0.0</a>.</em>

**Comment by psinghfmr** [comment]
Hi, Installed litmus on EKS using https://raw.githubusercontent.com/litmuschaos/litmus/2.6.0/mkdocs/docs/2.6.0/litmus-2.6.0.yaml. Now the self agent is in pending state. Still see only three running pods (frontend, server & mongo) but I guess scheduler pod should be running for self agent to connect but not happening. Can you help me out here? Thanks

**Reply by gdsoumya** [reply]
Can you see the graphql server logs, I think the deployment failed most probably due to some missing rbac permissions

**Reply by psinghfmr** [reply]
Yeah there are few pods not deployed due to constraints. I'm trying to get work around. Thanks
